x-pack/plugins/security_solution/public/timelines/components/timeline/body/column_headers/helpers.ts

```diff
@@ -23,17 +23,19 @@ export const getColumnHeaders = (
   headers: ColumnHeaderOptions[],
   browserFields: BrowserFields
 ): ColumnHeaderOptions[] => {
-  return headers.map((header) => {
-    const splitHeader = header.id.split('.'); // source.geo.city_name -> [source, geo, city_name]
+  return headers
```

Comment:
This is a fix for this error:
![Captura de Pantalla 2021-07-20 a les 11 20 41](https://user-images.githubusercontent.com/17747913/126304270-1991b277-d02e-4f50-8a0a-fde2a9aa9e68.png)
to reproduce:
- Open any saved timeline
- Open field browser
- Click reset fields

I replicated the same fix there is _x-pack/plugins/timelines/public/components/t_grid/body/column_headers/helpers.ts_